app/controllers/katello/api/v2/activation_keys_controller.rb

```diff
@@ -213,33 +213,28 @@ def remove_subscriptions
 
     api :PUT, "/activation_keys/:id/content_override", N_("Override content for activation_key")
     param :id, :number, :desc => N_("ID of the activation key"), :required => true
-    param :content_override, Hash, :desc => N_("Content override parameters"), :deprecated => true do
+    param :content_override, Hash, :desc => N_("Content override parameters") do
       param :content_label, String, :desc => N_("Label of the content"), :required => true
       param :value, String, :desc => N_("Override to a boolean value or 'default'"), :required => true
     end
```

Comment:
If this single `content_override` param can only be used for `enabled`, I think it would be good to make that more clear in the API docs, it doesn't seem clear to me